source/common/upstream/load_balancer_impl.h

```diff
@@ -88,35 +90,41 @@ class LoadBalancerBase : public LoadBalancer {
   // per_priority_health for that priority level, and may update per_priority_load for all
   // priority levels.
   void static recalculatePerPriorityState(uint32_t priority, const PrioritySet& priority_set,
-                                          PriorityLoad& priority_load,
-                                          PriorityLoad& degraded_priority_load,
-                                          std::vector<uint32_t>& per_priority_health,
-                                          std::vector<uint32_t>& per_priority_degraded);
+                                          HealthyLoad& priority_load,
```

Comment:
Mind here or in a  follow-up renaming priority_load, per_priority_load_ and friends to healthy_priority_load healthy_per_priority_load_ etc. just so the naming is parallel?